tiledb/sm/query/result_tile.cc

```diff
@@ -169,23 +170,19 @@ std::string ResultTile::coord_string(uint64_t pos, unsigned dim_idx) const {
   assert(!coord_tile_val.empty());
   auto cell_num = coord_tile_off.cell_num();
   auto val_size = coord_tile_val.size();
-
```

Comment:
Let's keep the original spaces in the function.

Reply:
yes, done that